Code/LauncherUnified/Launcher.cpp

```diff
@@ -26,6 +26,7 @@
 
 #include <CryLibrary.h>
 #include <ISystem.h>
+#include <ILevelSystem.h>
```

Comment:
Nit-pick: can we remove the #include as well?

Reply:
D'oh, fixed.